test/glue/pulp/consumer_test.rb

```diff
@@ -184,6 +184,10 @@ def test_uninstall_package_group
 
     def test_install_consumer_errata
       erratum_id = RepositorySupport.repo.errata_json.select { |errata| errata['id'] == 'RHEA-2010:0002' }.first['id']
+      profile = [{"vendor" => "FedoraHosted", "name" => "elephant",
+                  "version" => "0.3", "release" => "0.8",
+                  "arch" => "noarch", :epoch => ""}]
```

Comment:
i remember you saying that the lack of a package profile was causing pulp to error when performing this action, was an upstream issue opened for this?